src/doc/oiiotool.rst

```diff
@@ -2193,6 +2193,16 @@ current top image.
     image comprised of the subimages of all original images appended
     together.
 
+.. option:: --layersplit
+
+    Remove the top image from the stack, split it into its consituent
+    channel-name-based layers, and push them all onto the stack
+    (first to last).
```

Comment:
```suggestion
    Remove the top image from the stack, split it into a separate image for
    each of its constituentchannel-name-based layers, and push them all
    onto the stack (first to last).
```

Reply:
A spelling fix, and also added a clause to clarify that you are splitting into multiple images, not just subimages or whatever.

Reply:
Updated, thanks !